shopfloor_mobile/static/wms/src/components/packaging-qty-picker.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright 2020 Camptocamp SA (http://www.camptocamp.com)
  * @author Simone Orsi <simahawk@gmail.com>
- * Copyright 2021 Jacques-Etienne Baudoux (BCIM)
+ * Copyright 2021-2022 Jacques-Etienne Baudoux (BCIM)
```

Comment:
there's no need to change the (C) year...

Reply:
was fixed in https://github.com/OCA/wms/pull/485 ...

Reply:
fixed here also now